shell/nginx.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+
+echo "server {
+          server_name $1;
+          root $2/web;
+
+          location / {
+              # try to serve file directly, fallback to app.php
+              try_files \$uri /app.php\$is_args\$args;
+          }
+
+          location ~ ^/(app|app_dev|config)\.php(/|$) {
+              fastcgi_pass unix:/var/run/php5-fpm.sock;
+              fastcgi_split_path_info ^(.+\.php)(/.*)$;
+              include fastcgi_params;
+              fastcgi_param SCRIPT_FILENAME \$document_root\$fastcgi_script_name;
+              fastcgi_param HTTPS off;
+              fastcgi_read_timeout 600;
+          }
+
+          error_log /var/log/nginx/$1.error.log;
+          access_log /var/log/nginx/$1.access.log;
+      }" >> /etc/nginx/sites-available/symfony
```

Comment:
I can to add. But in this there is no sense. In one box we have only one nginx config